spread/qemu.go

```diff
@@ -97,6 +99,16 @@ func systemPath(system *System) string {
 	return os.ExpandEnv("$HOME/.spread/qemu/" + system.Image + ".img")
 }
 
+func ovmfPath() string {
+	// this will work on ubuntu/debian/fedora
+	ovmfPath := "/usr/share/OVMF/OVMF_CODE.fd"
+	// allow packaging to override default
+	if p, err := os.Readlink("/usr/share/spread/qemu-efi-firmware"); err == nil {
```

Comment:
Tests of this will be affected by the host.

Reply:
Nice catch, fixed now.